sentinel5dl/__init__.py

```diff
@@ -74,13 +84,36 @@ def __http_request(path, filename=None):
     logger.debug(f'Requesting {url}')
     with open(filename, 'wb') if filename else io.BytesIO() as f:
         curl = pycurl.Curl()
+        if ca_info:
+            curl.setopt(pycurl.CAINFO, ca_info)
         curl.setopt(curl.URL, url.encode('ascii', 'ignore'))
 
         curl.setopt(curl.USERPWD, f'{USER}:{PASS}')
         curl.setopt(curl.WRITEDATA, f)
         curl.setopt(curl.FAILONERROR, True)
-        curl.perform()
+
+        # abort if data trnsfer is slower than 30 bytes/sec during 60 seconds
+        curl.setopt(pycurl.LOW_SPEED_TIME, 10)
+        curl.setopt(pycurl.LOW_SPEED_LIMIT, 300)
```

Comment:
Does it make sense to fail on slow connections? Wouldn't we just retry and fail again? It's unlikely that the speed changes between try and re-try, isn't it?

Reply:
the idea of this is to retry if the download connection is not responding but didn't errored